### PR TITLE
AMS-3: Make create button in select view a form extension

### PR DESCRIPTION
**Pull request by willy-ahva**
Create a form extension for the footer button in the view selector.
With this we are able to extend this button add the behavior we need.

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | no need
| Added Behats                      | no need
| Changelog updated                 | no need
| Review and 2 GTM                  |
| Micro Demo to the PO (Story only) | no update on UI
| Migration script                  |
| Tech Doc                          |

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml [hunk 1]
```diff
@@ -337,6 +337,7 @@ config:
         # Grid view selector
         pim/grid/view-selector: pimenrich/js/grid/view-selector
         pim/grid/view-selector-footer: pimenrich/js/grid/view-selector-footer
+        pim/grid/view-selector-footer-create: pimenrich/js/grid/view-selector-footer-create
```
Comment (grena) [inline src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml]: What do you think to change the name by
```
pim/grid/view-selector-footer/create-view
```
?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/grid/view-selector-footer-create.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+/**
+ * Footer extension for the Datagrid View Selector.
+ *
+ * Contains a "create" button to allow the user to create a view from the current
+ * state of the grid (regarding filters and columns).
+ *
+ * @author    Adrien Petremann <adrien.petremann@akeneo.com>
```
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/grid/view-selector-footer-create.js]: done

File: src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml [hunk 3]
```diff
@@ -442,6 +443,7 @@ config:
         # Grid view selector
         pim/template/grid/view-selector: pimenrich/templates/grid/view-selector.html
         pim/template/grid/view-selector-footer: pimenrich/templates/grid/view-selector-footer.html
+        pim/template/grid/view-selector-footer-create: pimenrich/templates/grid/view-selector-footer-create.html
```
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml]: done

File: src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml (hunk 1 above)
Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/config/requirejs.yml]: done

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/grid/view-selector-footer-create.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+                $input.on('input', function () {
+                    label = $input.val();
+                    if (!label.length) {
+                        $submitBtn.hide();
+                    } else {
+                        $submitBtn.show();
+                    }
+                }).on('keypress', function (e) {
+                    if ((e.keyCode || e.which) === 13 && label.length) {
```
Comment (aRn0D) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/grid/view-selector-footer-create.js]: yoda style

Comment (willy-ahva) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/grid/view-selector-footer-create.js]: done

**Comment by willy-ahva** [comment]
1 build green before the yoda style fix
then 2 builds with 2 different "unstable"
I merge

**Pull request merged**